pro/src/screens/Bookings/BookingsRecapTable/components/Table/Body/CollectiveBookingDetails/CollectiveBookingDetails.module.scss

```
@@ -10,31 +10,76 @@
 .details-column {
   flex-basis: 50%;
   flex-grow: 1;
+
+  &:last-child {
+    display: flex;
+    justify-content: flex-end;
+  }
 }
 
 .contact-details-title {
   @include fonts.button;
+  margin-bottom: rem.torem(8px);
 }
 
 .contact-details {
   display: flex;
+  width: rem.torem(315px);
   flex-direction: column;
-  gap: rem.torem(16px);
+  gap: rem.torem(8px);
   padding: rem.torem(16px);
   background-color: colors.$white;
-  box-shadow: 0 3px 4px rgb(103 104 107 / 10%);
   border-radius: rem.torem(6px);
+  border: rem.torem(1px) solid #cbcdd2;
 }
 
 .contact-detail {
   display: flex;
   gap: rem.torem(8px);
 }
```

C'est un détail mais c'est possible de faire comme ça aussi
```suggestion
  &-icon {
    display: inline-flex;
    align-self: center;

    path {
      fill: colors.$black;
    }
  }
}
```

---

oui merci c'est plus propre